src/api/entitlements.js

```diff
@@ -235,11 +235,13 @@ export class Entitlements {
 
   /**
    * A 3p site should call this method to consume a Google metering entitlement.
-   * When a metering entitlement is consumed, that depletes the user's
-   * remaining "free reads".
+   * When a metering entitlement is consumed, SwG shows the user a metering dialog.
+   * When the user closes the dialog, SwG depletes one of the user's remaining
+   * "free reads".
+   * @param {!Function=} onCloseDialog Called after the user closes the dialog.
```

Comment:
Can we let this be null? Do we expect something to be called after every dialog flow?

Reply:
Good call, the function is now optional

Reply:
Please update the comment above and elsewhere to be "?Function=" instead of"!Function="

Reply:
I think `!Function=` is ideal because it tells the Closure compiler "Either a Function or undefined are OK". Whereas `?Function=` would tell Closure "Either a null, or a function, or undefined are OK". In this case, I think just expecting a Function or undefined would make more sense

Reply:
Then again, allowing `null` wouldn't hurt. I'll update the code :)